changelog.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v31.1
+
```

Comment:
Not sure if we need this in release patch (there is a `no-changelog` label). Otherwise, the PR# would be 3973?